ui-tests/src/test/java/org/openmrs/reference/page/RegistrationPage.java

```diff
@@ -343,5 +350,24 @@ public String getSimilarPatientName() {
     public String getSimilarPatientInfo() {
         return findElement(By.cssSelector("#similarPatientsSelect .info")).getText();
     }
+    
+    public void SelectRelationshipType() {
+    	clickOn(SELECT_RELATIONSHIP_TYPE);
+      Select relationshipType = new Select(driver.findElement(By.id("relationship_type")));
+    	relationshipType.selectByVisibleText("Doctor");
+    	setTextToFieldNoEnter(PERSON_NAME, NAME);
```

Comment:
Do you think `selectRelationshipType()` code base is well indented ?

Reply:
Sure this is well convention ed. Thanks.

Reply:
Have you had a look at its current indention [here](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/37a05893a9e5141db58d297c7e26f0fcaba506da/ui-tests/src/test/java/org/openmrs/reference/page/RegistrationPage.java#L354) ?

Reply:
> Have you had a look at its current indention [here](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/37a05893a9e5141db58d297c7e26f0fcaba506da/ui-tests/src/test/java/org/openmrs/reference/page/RegistrationPage.java#L354) ?

Hopefullly the recent commit address that.

Reply:
Thanks @kdaud  for the review